lib/ente_theme_data.dart

```diff
@@ -83,9 +80,18 @@ ElevatedButtonThemeData buildElevatedButtonThemeData(
       fontFamily: 'Inter-SemiBold',
       fontSize: 18,
     ),
-    padding: EdgeInsets.symmetric(vertical: 14),
+    padding: EdgeInsets.symmetric(vertical: 18),
     shape: const RoundedRectangleBorder(
       borderRadius: BorderRadius.all(Radius.circular(8)),
     ),
   ));
 }
+
+TextStyle gradientButtonTextTheme() {
+  return TextStyle(
+    color: Colors.white,
```

Comment:
Will we need to change the color for the black and white theme? 
if yes, should we change this to Theme.of(context).textTheme.headline6.copyWith(..)?